ttnn/ttnn/core.py

```diff
@@ -43,6 +43,13 @@ def has_tile_padding(tensor):
     return False
 
 
+def dump_stack_trace_on_segfault():
```

Comment:
nice! does this make sense to expose as a debug setting here: `ttnn/ttnn/__init__.py`

Reply:
I don't know that we need to, but what are you thinking?

Reply:
Yeah that is a good idea.  @arakhmati ?